src/code/z_sound_source.c

```diff
@@ -0,0 +1,75 @@
+#include "global.h"
```

Comment:
Oh, I suppose this is also missing a file description, but I imagine the whole lot is going to get a second pass anyway.